code/modules/vehicles/scooter.dm

```diff
@@ -0,0 +1,105 @@
+/obj/vehicle/scooter
+	name = "scooter"
+	desc = "A fun way to get around."
+	icon_state = "scooter"
+
+/obj/vehicle/scooter/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/wrench))
+		user << "<span class='notice'>You begin to remove the handlebars...</span>"
+		if(do_after(user, 40/I.toolspeed, target = src))
+			new /obj/vehicle/scooter/skateboard(get_turf(src))
+			new /obj/item/stack/rods(get_turf(src),2)
+			user << "<span class='notice'>You remove the handlebars from [src].</span>"
+			playsound(get_turf(user), 'sound/items/Ratchet.ogg', 50, 1)
+			qdel(src)
+
+/obj/vehicle/scooter/handle_vehicle_layer()
+	if(dir == SOUTH)
+		layer = MOB_LAYER+0.1
+	else
+		layer = OBJ_LAYER
+
+/obj/vehicle/scooter/handle_vehicle_offsets()
+	..()
+	if(buckled_mobs.len)
+		for(var/m in buckled_mobs)
+			var/mob/living/buckled_mob = m
+			switch(buckled_mob.dir)
+				if(NORTH)
+					buckled_mob.pixel_x = 0
+					buckled_mob.pixel_y = 4
+				if(EAST)
+					buckled_mob.pixel_x = -2
+					buckled_mob.pixel_y = 4
+				if(SOUTH)
+					buckled_mob.pixel_x = 2
+					buckled_mob.pixel_y = 4
+				if(WEST)
+					buckled_mob.pixel_x = 0
+					buckled_mob.pixel_y = 4
```

Comment:
note to self: switch values of south/west directions, they are swapped for some reason